src/cuda/common.cu

```diff
@@ -26,15 +26,18 @@ using std::max;
 // a , f are intermediate results from function onedim_fseries_kernel_precomp()
 // (see cufinufft/contrib/common.cpp for description)
 template<typename T>
-__global__ void fseries_kernel_compute(int nf1, int nf2, int nf3, T *f,
-                                       cuDoubleComplex *a, T *fwkerhalf1, T *fwkerhalf2,
-                                       T *fwkerhalf3, int ns) {
+__global__ void fseries_kernel_compute(int nf1, int nf2, int nf3, T *f, T *a,
+                                       T *fwkerhalf1, T *fwkerhalf2, T *fwkerhalf3,
+                                       int ns) {
   T J2  = ns / 2.0;
   int q = (int)(2 + 3.0 * J2);
   int nf;
-  cuDoubleComplex *at = a + threadIdx.y * MAX_NQUAD;
-  T *ft               = f + threadIdx.y * MAX_NQUAD;
+  T *at = a + threadIdx.y * MAX_NQUAD;
+  T *ft = f + threadIdx.y * MAX_NQUAD;
   T *oarr;
+  // standard parallelism pattern in cuda. using a 2D grid, this allows to leverage more
```

Comment:
good comments... otherwise no reader would figure out what the threadIdx.y is doing!